### DISCUSSION TITLE
Anti Drag functionality

### DISCUSSION THREAD

**Discussion by RsFoe** [runelite/runelite#15186] [Ideas]
Could we have the Cox chests/seed vault added into the anti drag plugin. Right now the cox private chest is unbearable to left click gear out of. Maybe have toggles for overworld/cox/bank so people who dont want to use anti drag for normal gameplay can still have it for cox/bank.

**Comment by jari-65** [comment]
anti drag plugin is not working anymore. my mouse often moves when clicking bank or fighting even in guthix. hoping it start working or feature is added to permanent plugins. 

example hard to eat  while fighting because it start moving food in inventory

**Reply by cdfisher** [reply]
This is unrelated to the original post, and Anti Drag is already part of core RuneLite.

**Reply by jari-65** [reply]
ok thank you. still problem in my playing style. found it and it looks that it's working
